spec/proxy.spec.js

```diff
@@ -567,7 +567,7 @@ describe('proxy', function() {
     }
 
     function requestFake(req) {
-        const responseStatus = req.headers['x-give-response-status'] || 200;
+        const responseStatus = parseInt(req.headers['x-give-response-status'] || "200");
```

Comment:
Good catch, just a tiny nitpick - could this be `const responseStatus = parseInt(req.headers['x-give-response-status']) || 200` instead?

Reply:
Fixed.